public/app/core/services/echo/backends/grafana-javascript-agent/GrafanaJavascriptAgentBackend.ts

```diff
@@ -74,6 +74,7 @@ export class GrafanaJavascriptAgentBackend
       batching: {
         sendTimeout: 1000,
       },
+      internalLoggerLevel: options.internalLoggerLevel || defaultInternalLoggerLevel,
```

Comment:
It's official part of SDK ([more docs here](https://github.com/grafana/faro-web-sdk/blob/main/docs/sources/tutorials/quick-start-browser.md#how-to-activate-debugging)). It doesn't change the behavior but allows additional debug messages on what's going on under the hood (e.g. allows you to see when logs were sent out, or if a log was skipped because it's a duplicate).